src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -206,14 +208,26 @@ public PrivilegesEvaluatorResponse evaluate(final User user, String action0, fin
             action0 = PutMappingAction.NAME;
         }
 
+        final PrivilegesEvaluatorResponse presponse = new PrivilegesEvaluatorResponse();
+
         final TransportAddress caller = threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_REMOTE_ADDRESS);
-        final Set<String> mappedRoles = (injectedRoles == null) ? mapRoles(user, caller) : injectedRoles;
+        Set<String> mappedRoles = (injectedRoles == null) ? mapRoles(user, caller) : injectedRoles;
+        final String assumeRolesString = threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_ROLES_VALIDATION);
```

Comment:
Please remove assume role references from everywhere (file name, constants, variable, error message and comments) .